src/js/store/configureStore.js

```diff
@@ -0,0 +1,16 @@
+var createStore  = require('redux').createStore;
+var rootReducer = require('../reducers');
+
+var configureStore = function(initialState) {
+  var store = createStore(rootReducer, initialState);
+  if (module.hot) {
+    // Enable Webpack hot module replacement for reducers
+    module.hot.accept('../reducers', () => {
+      var nextReducer = require('../reducers').default;
+      store.replaceReducer(nextReducer)
+    })
```

Comment:
ok, I only noticed this convention in a couple examples.  we can add it later if we need it :) :+1: